docs/docs-content/getting-started/cluster-profiles.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_label: "Cluster Profiles"
+title: "Cluster Profiles"
+description: "Spectro Cloud Palette Cluster Profiles"
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 30
+tags: ["getting-started"]
+---
+
+Cluster profiles are composed of layers using packs, Helm charts, and custom manifests to meet specific types of
```

Comment:
This is a great overview but what is that makes cluster profiles so useful? Maybe it's their delerative nature that allows users to visually see and understand what makes up a k8s cluster? We are missing the content that explains why this concept is so powerful to the user. We answer the what, but not the why.

One thing to consider, take a peek at the [slide deck template icons](https://docs.google.com/presentation/d/1b8d67gEKXDLRL6G0dGtrBgMe3kXOkqcL/edit#slide=id.p94), specifically slide 72 - 79 maybe that's a better visual to help articulate the value by seeing more layers with examples?

Reply:
I did some work on this page as directed. I've kept the diagram which shows the different kinds of cluster profiles, but added a diagram from slide 79 in the page under the Packs section as it showcases lots of different technologies.